scripts/pi-hole/js/queries.js

```diff
@@ -136,7 +138,8 @@ $(document).ready(function() {
     });
     $("#all-queries tbody").on( "click", "button", function () {
         var data = tableApi.row( $(this).parents("tr") ).data();
-        if (data[4] === "Pi-holed")
+        status = data[4];
+        if (status.substr(0,2) === "Pi")
```

Comment:
Since you can't whitelist a wildcard-blocked domain, we should not show the whitelist button. Same idea on the Core repo with the block page.